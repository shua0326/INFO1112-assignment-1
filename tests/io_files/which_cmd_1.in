which ls no_exist cd pwd grep
