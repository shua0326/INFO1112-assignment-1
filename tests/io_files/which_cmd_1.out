>> /usr/sbin/ls
no_exist not found
cd: shell built-in command
pwd: shell built-in command
/usr/sbin/grep
>> 
